applications/ShapeOptimizationApplication/python_scripts/response_functions/packaging_response_base.py

```diff
@@ -11,9 +11,9 @@
 import time as timer
 import KratosMultiphysics as KM
 from KratosMultiphysics import Logger
-from .response_function import ResponseFunctionBase
+from KratosMultiphysics.response_functions.response_function_interface import ResponseFunctionInterface
```

Comment:
Looks a bit strange to me, but I guess this is the way to go

Reply:
you mean the long import? Yes that's how it has to be done